apps/namada-interface/src/slices/AccountsNew/actions.ts

```diff
@@ -34,13 +63,27 @@ type ShieldedKeysAndAddressesWithNewAccountDetails = {
  */
 export const createShieldedAccount = createAsyncThunk<
   ShieldedKeysAndAddressesWithNewAccountDetails | undefined,
-  NewAccountDetailsWithPassword
+  NewAccountDetailsWithPassword & { notify?: boolean }
 >(
   ADD_ACCOUNT_TO_LEDGER,
-  async (newAccountDetails: NewAccountDetailsWithPassword) => {
+  async (
+    newAccountDetails: NewAccountDetailsWithPassword & { notify?: boolean },
+    thunkAPI
+  ) => {
+    const { chainId, alias, password, notify } = newAccountDetails;
+
+    if (notify) {
```

Comment:
I do not really like this notify flag. Added it because `createShieldedAccount` is used for both adding new accounts and loading initial ones. I think the best solution would be to split those actions into two.